### PR TITLE
Converted topic on manual PSC registration to updating PSC credentials

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/1162.

@anchal-agrawal is this accurate? Thanks!

### CODE REVIEW

**Comment by andrewtchin** [comment]
Yes that's fine since there haven't been any updates to that file yet. I'll ping you if that file is changed.

**Comment by stuclem** [comment]
Thanks @andrewtchin

File: docs/user_doc/vic_vsphere_admin/manual_register_psc.md [hunk 1]
```diff
@@ -1,34 +1,25 @@
...
 
-The vSphere Integrated Containers appliance deploys successfully, but the initialization of the appliance fails.
+When you deploy the vSphere Integrated Containers appliance, you must register it with the VMware Platform Services Controller that manages the vCenter Server instance on which you deployed the appliance.
 
-## Problem ##
+If the credentials for the Platform Services Controller change after you deployed the appliance, the appliance can no longer connect to vCenter Server. In this case, you must update the credentials with which the appliance is registered with the Platform Services Controller.
 
-When you enter the vCenter Server credentials at the login prompt on the vSphere Integrated Containers Getting Started page, you see a red alert with the message `Failed to register with PSC. Please check the vSphere user domain PSC settings and try again`. 
+**NOTE**: To update the registration, you must log into the appliance VM and run commands manually. You cannot update the registration by using the Re-Initialize button on the vSphere Integrated Containers Getting Started page at https://<i>vic_appliance_address</i>.
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_vsphere_admin/manual_register_psc.md]: @andrewtchin Now that we restart VIC services every time the `Re-Initialize` button is used, we should be able to update PSC credentials by simply using that button, correct?

Comment (anchal-agrawal) [reply docs/user_doc/vic_vsphere_admin/manual_register_psc.md]: cc @morris-jason

Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/manual_register_psc.md]: @andrewtchin @anchal-agrawal @morris-jason bear in mind that this update applies to 1.3.x. If the Re-Initialize behaviour was changed in 1.4, then it is not relevant here. Thanks!

Comment (stuclem) [reply docs/user_doc/vic_vsphere_admin/manual_register_psc.md]: Thanks @andrewtchin and @anchal-agrawal. I'll merge this one, and create a new issue to update the behaviour in 1.4.

**Pull request merged**